src/controllers/adminContoller.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
Super admin should not be created, update or deleted, should be seeded in the database at the time of up deployment. super admin is a user who has higher permission. For creating, deleting, updating or fetching should uses methods in the user controller, just modify those functions a little bit to ensure that admin can also perform those operations on any user.